generator/uscore/uscore_generator.rb

```diff
@@ -583,40 +587,31 @@ def create_must_support_test(sequence)
           description: %(
             US Core Responders SHALL be capable of populating all data elements as part of the query results as specified by the US Core Server Capability Statement.
             This will look through the #{sequence[:resource]} resources found previously for the following must support elements:
-          )
-        }
 
-        sequence[:must_supports][:elements].each do |element|
-          test[:description] += %(
-            * #{element[:path]})
-          # class is mapped to local_class in fhir_models. Update this after it
-          # has been added to the description so that the description contains
-          # the original path
-          element[:path] = element[:path].gsub(/(?<!\w)class(?!\w)/, 'local_class')
-        end
+            #{must_support_list.join("\n            ")}
 
-        must_support_extensions = sequence[:must_supports][:extensions]
-        must_support_extensions.each do |extension|
-          test[:description] += %(
-            * #{extension[:id]})
-        end
 
+            For elements of type 'reference' with one or more target profiles from US Core, this test will ensure that at least one of each resource type
```

Comment:
Should this only be included for the tests which actually have to perform this checking?

Reply:
👍